index.js

```diff
@@ -238,7 +240,8 @@ client.on(Events.InteractionCreate, async (interaction) => {
         // Send task to Marvin
         const errorOccured = sendTask(mailOptions);
 
-        errorOccured ? interaction.reply('Error sending email') : interaction.reply('Email sent');
+        const responseMessage = errorOccured ? 'Error sending email' : 'Email sent';
```

Comment:
Oh nice, think this was one of the comments from the previous PR. Great to see you taking in all the feedback and implementing it so quickly! Makes life super easy as a reviewer! 😄